testdata/serviceconfig.yml

```diff
@@ -11,28 +11,28 @@ rules:
     adapter: prometheus
     params:
       metrics:
-      - descriptorName: request_count
+      - descriptor_name: request_count
         # we want to increment this counter by 1 for each unique (source, target, service, method, response_code) tuple
         value: "1"
         labels:
           source: source.name | "unknown"
           target: target.name | "unknown"
           service: api.name | "unknown"
           method: api.method | "unknown"
-          response_code: response.http.code | 200
-      - descriptorName:  request_latency
-        value: response.latency | "0ms"
+          response_code: response.code | 200
+      - descriptor_name:  request_latency
+        value: response.latency
```

Comment:
Casualty of my attempt to swap to jsonpb parsing instead of json parsing. Added it back.